babel.config.js

```diff
@@ -0,0 +1,7 @@
+// used for flow type stripping and
+module.exports = function(api) {
+  api.cache(false);
```

Comment:
why are we disabling the cache here? Please could you document if this is required.